awscli/data/codebuild/2016-10-06/completions-1.json

```diff
@@ -1,14 +1,114 @@
 {
   "version": "1.0",
   "resources": {
+    "BuildBatche": {
```

Comment:
This seems like the script is not able to handle plurals that end in `es`. I wonder if this is something easy to fix?